lib/core-integration/src/Test/Integration/Faucet.hs

```diff
@@ -2239,6 +2250,20 @@ maryIntegrationTestAssets = maMnemonics >>= take 3
         ]
     combined p = simple p `TokenBundle.add` fruit p
 
+    seaHorse is p = bundle p $ flip map is $ \i ->
+        (UnsafeTokenName ("SeaHorse" <> B8.pack (show i)), TokenQuantity 1)
+
+    specialAssets =
+        flip zip (map (\is -> mint (seaHorse is) (maryAssetScripts !! 4)) (chunks 2 [1 .. 200]))
```

Comment:
`maryAssetScripts !! 4` looked like the next un-used one, but should have some nicer way of choosing one.

chunks of 5 assets per address were to big (think we fund 10 or 20 addresses per tx), so went with 2

Reply:
Fixed - moved it to a separate definition